src/codeflare_sdk/cluster/cluster.py

```diff
@@ -71,6 +74,11 @@ def __init__(self, config: ClusterConfiguration):
         self.app_wrapper_yaml = self.create_app_wrapper()
         self._job_submission_client = None
         self.app_wrapper_name = self.config.name
+        if (
+            "PYDEVD_IPYTHON_COMPATIBLE_DEBUGGING" in os.environ
+            or "JPY_SESSION_NAME" in os.environ
+        ):  # If running Jupyter NBs in VsCode or RHOAI/ODH display UI buttons
+            self._cluster_up_down_buttons()
```

Comment:
Can you pull the if statement into a utility function so it becomes like:
```suggestion
        if running_in_jlab():
            self._cluster_up_down_buttons()
```
We'll end up needing this in a lot of places for the new functionality

Reply:
Good idea will update :+1: